flink-ml-lib/src/main/java/org/apache/flink/ml/evaluation/binaryclassification/BinaryClassificationEvaluator.java

```diff
@@ -362,7 +234,7 @@ public void mapPartition(
                 reduceMetrics = reduceMetrics.merge(iter.next());
             }
             Map<String, Double> map = new HashMap<>();
-            map.put(AREA_UNDER_ROC, reduceMetrics.areaUnderROC);
+            map.put(AREA_UNDER_ROC, 1. - reduceMetrics.areaUnderROC);
```

Comment:
Here, the variable name 'reduceMetrics.areaUnderROC' is not suitable. Maybe 'reduceMetrics.areaAboveROC' is better.

Reply:
Good point! I've changed the way to calculate areaUnderROC, so there is no more `1. - ` here.